docs/src/guides/setup-dev.md

```diff
@@ -1,68 +1,21 @@
-# Installing dependencies
+# Installing development setup for ZKsync
 
 ## TL;DR
 
-This is a shorter version of setup guide to make it easier subsequent initializations. If it's the first time you're
+This is a shorter version of setup guide to make it easier for initializations. If it's the first time you're
 initializing the workspace, it's recommended that you read the whole guide below, as it provides more context and tips.
 
-If you run on 'clean' Ubuntu on GCP:
+Just run the following command on Debian-based Linux:
 
 ```bash
-# For VMs only! They don't have SSH keys, so we override SSH with HTTPS
-git config --global url."https://github.com/".insteadOf git@github.com:
-git config --global url."https://".insteadOf git://
```

Comment:
These lines are important for VMs, because submodule fetching may break if running on a fresh machine without valid ssh keys; however they're missing in the script.

Reply:
The standard now really is to declare submodules with https, like we [do](https://github.com/matter-labs/zksync-era/blob/main/.gitmodules). I did follow the tree of submodules down and it seems all of them use https.
Do you think it's worth it to keep this, since it's such a small edge case?